app/Helpers/util/string.php

```diff
@@ -29,6 +29,10 @@ function attributeEscape(?string $input): string
 
 function isValidUsername(?string $username): bool
 {
+    if (Str::startsWith($username, '__')) {
+        return true;
+    }
```

Comment:
make sure that seeded users can be used and accessed

Reply:
The `CtypeAlnum` validator used by the API (and the registration page) doesn't allow underscores:
```
$ curl "localhost/API/API_GetUserSummary.php?y=engineer-secret&u=__administrator"

{"message":"The u must only contain unaccented letters and numbers.",
 "errors":{"u":["The u must only contain unaccented letters and numbers."]}}
```
It will also choke on the dashes in things like `release-manager`.

For a purely seeded database, why can't we just use PascalCase? `Administrator`, `ReleaseManager`, etc.

Reply:
I think I initially added those prefixes to not run into conflicts with an existing imported dump but that's not how it should be used in the first place.
I'll remove those and make sure that the "safe" variants are used for usernames to make them work with our validation rules - thanks for finding that!

Reply:
Went with your suggestion to use PascalCase: https://github.com/RetroAchievements/RAWeb/pull/1770/commits/90627af82e73f4be7f15940613d84cd4b2a580c4